### ISSUE TITLE
Missing fonts on login screen on Ubuntu 18.04 on Azure

### ISSUE THREAD

**Issue by oybektoirov** [neutrinolabs/xrdp#1641]
Missing fonts on login screen on Ubuntu 18.04 LTS on Azure. Connecting from Mac using Microsoft Remote Desktop version 10.3.12 (1801).

![](https://user-images.githubusercontent.com/4557659/44992138-aef44780-af96-11e8-8da2-2aa99e28dabc.png)

I ran these commands below from Microsoft RDP [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/use-remote-desktop) to get RDP setup:
```
sudo apt-get update
sudo apt-get -y install xfce4
sudo apt-get -y install xrdp
sudo systemctl enable xrdp
echo xfce4-session >~/.xsession
sudo service xrdp restart
```

Versions:
xrdp version 0.9.5-2
Microsoft RDP Version 10.3.12 (1801)
MacOS Catalina 10.15.3

**Comment by oybektoirov** [comment]
I also ran sudo apt install ubuntu-desktop and restarted xrdp but didnt help

**Comment by metalefty** [comment]
Dup of #643. Fixed at #1235.

**Comment by quang777** [comment]
Question, this was reported broken in Ubuntu 18.04 w/ xrdp version 0.9.5. The fix per #1235 is 0.9.9, but you can't get that version on Ubuntu 18.04 LTS. How are 18.04 users supposed to get the fix? Is it possible to back port this fix or make it available for 18.04 repos? I thought the idea of LTS releases were extended updates? Or is this more of an Debian/Ubuntu issue? I'm not familiar with the two different development pipelines of the OS vs package maintainers.

**Comment by metalefty** [comment]
>  Is it possible to back port this fix or make it available for 18.04 repos? 

Yes, it is possible but it's up to Ubuntu/Debian team. Report the issue to Ubuntu team is the first step to get the fix backported.

**Comment by quang777** [comment]
Will do, I started a thread on Launchpad.net. Thanks!
